examples/activities/sleep_activity.rb

```diff
@@ -1,11 +1,10 @@
 class SleepActivity < Temporal::Activity
   timeouts(
-    schedule_to_close: 5,
-    schedule_to_start: 5,
-    start_to_close: 5,
-    heartbeat: 5
```

Comment:
What is the reason for removing these?

Reply:
It doesn't heartbeat, and I wasn't sure what the point of having a `schedule_to_start` timeout for this particular workflow was.
My main goal was to increase the timeout so that a larger sleep amount could be passed in.

Reply:
Got it 👍